README.md

```diff
@@ -6,6 +6,7 @@
 
 #### Github Actions 🤖
 - [Brian Douglas](https://github.com/bdougie/bdougie)
+- [Shawn Wang](https://github.com/sw-yx/sw-yx) 
```

Comment:
@filiptronicek bro, I mean leave a line after it, because the next category is just after it na 😃
and sorry my mistake, i said space instead of a line 😅